### DISCUSSION TITLE
Rename "Properties (ExplorerPatcher)" shortcut in Start menu?

### DISCUSSION THREAD

**Discussion by chip33** [valinet/ExplorerPatcher#870] [Ideas]
The start menu shortcut is a good idea but as many people search the start menu alphabetically would it be more intuitive and easily found if the shortcut was named ExplorerPatcher?
People would be expecting this to open the Properties menu as default behaviour anyway.
I renamed it myself for convenience but it gets renamed on an update.

**Comment by valinet** [comment]
The name is localized. I thought about "ExplorerPatcher Properties", but that may look weird in other languages (order of nouns, whatever). You can still search for "ExplorerPatcher" and that works.
